site/dev/common.sh

```diff
@@ -69,7 +69,7 @@ install_deps () {
   echo " --> install deps"
 
   # Use pip to install or upgrade dependencies from the 'requirements.txt' file quietly
-  pip -q install -r requirements.txt --upgrade
+  pip3 -q install -r requirements.txt --upgrade
```

Comment:
`pip3` probably works better for most env.

`python3` is also used at line 204